src/OPL3.cpp

```diff
@@ -163,10 +178,10 @@ void OPL3::setChannelRegister(byte baseRegister, byte channel, byte value) {
  * @param operatorNum - The operator to change [0, 1].
  * @param value - The value to write to the operator's register.
  */
-void OPL3::setOperatorRegister(byte baseRegister, byte channel, byte operatorNum, byte value) {
+void OPL3::setOperatorRegister(short baseRegister, byte channel, byte operatorNum, byte value) {
```

Comment:
Same here.

Reply:
same here, I borrowed it from the other class. Fixed in new force-pushed commit